server/pkg/git/integrate.go

```
@@ -0,0 +1,183 @@
+	}
+	state, err := Status(repo, checkout)
+	if err != nil {
+		return "", "", err
+	}
+	ours = state.Branch
+	if ours == "" {
+		ours = "HEAD"
+	}
+	gd, err := gitDir(dir)
+	if err != nil {
+		return ours, "", nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 100) but it returns nil

(nilerr)